tests/model/test_model.py

```diff
@@ -535,11 +535,10 @@ def test_fail_get_messages(self, mocker, error_response,
     ], ids=['muting_205', 'unmuting_205', 'first_muted_205',
             'last_unmuted_205'])
     def test_toggle_stream_muted_status(self, mocker, model,
-                                        initial_muted_streams, value):
+                                        initial_muted_streams, value,
+                                        response={'result': 'success'}):
```

Comment:
Is this to overcome pytest's caching issue or just to reuse the `dict`?

Reply:
Just for reuse